templates/default/rdrs-start.sh.erb

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env sh 
+
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:<%= node['mysql']['base_dir'] %>/lib/
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:<%= node['mysql']['base_dir'] %>/lib/private/
+
+<%= node['mysql']['base_dir'] %>/bin/rdrs -config=<%= node['ndb']['root_dir'] %>/rdrs_config.json  2>&1 > <%= node['ndb']['log_dir'] %>/rdrs_out.log &
```

Comment:
Does rdrs print to stdout all logs? If this log contains only some early logs it can be ok-ish but if it contains all logs that's a problem. On every restart the last logs will be lost.

Reply:
nothing is printed to stdout/stderr. I capture it just in case if there are stray printf statements or some third-party lib print to stdout. 
changed > to >>